libs/features/mas/src/settings.js

```diff
@@ -28,6 +29,7 @@ function getLocaleSettings({
 }
 
 function getSettings(config = {}) {
+    const ffDefaultflag = getMetadata(FF_DEFAULT_FLAG) === 'true';
```

Comment:
please move metadata logic to merch.js

Reply:
@bozojovicic you just call `getParameter(FF_DEFAULT_FLAG)` , it will work